extension-manifest-v3/src/background/cosmetics.js

```diff
@@ -205,13 +199,13 @@ async function adblockerOnMessage(msg, sender, sendResponse) {
     sendResponse({
       active: specificResponses.map((r) => r.active).some((a) => a),
       extended: specificResponses.map((r) => r.extended).flat(),
-      styles: '',
+      scripts: specificResponses.map((r) => r.scripts).flat(),
```

Comment:
Unless I'm missing something, scripts are intentionally not returned, since they are executed with "chrome.scripting" (on navigation events)

Reply:
See my comment https://github.com/ghostery/ghostery-extension/pull/916#discussion_r1006790485

Reply:
lets move this discussion to the other thread